styles.css

```diff
@@ -589,3 +589,105 @@ ul.list-items.skill-items {
 .button-5 {
   padding-left: 10px;
 }
+
+form {
+  display: block;
+  margin: 0;
+}
+
+.form-class {
+  display: flex;
+  flex-direction: column;
+  margin-top: 12px;
+  align-items: center;
+}
+
+.form-btn {
+  color: #6070ff;
+  background-color: #fff;
+  border-radius: 8px;
+  padding: 12px 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: 1px;
+  margin-top: 12px;
+  cursor: pointer;
+}
+
+.contact-info {
+  display: flex;
+  align-items: center;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 327px;
+  height: 48px;
+  margin-top: 12px;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+}
+
+.text-input {
+  font-family: "poppins", sans-serif;
+  font-size: 17px;
+  font-weight: 400px;
+  width: 327px;
+  height: 170px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin-top: 12px;
+}
+
+.Contact-paragraph {
+  color: #ebebff;
+  margin-top: 12px;
+  font-style: normal;
```

Comment:
- [ ] Please make your title bigger as in the figma design. Also, apply the exact background image as you can see below, your shape is bigger than the Figma design:
yours:
![image](https://user-images.githubusercontent.com/91486791/220392667-3fe01eac-13c8-4340-bc79-27ee07efc0e8.png)
figma:
![image](https://user-images.githubusercontent.com/91486791/220392781-1903395d-5f31-4ad6-af4e-1628b81af673.png)